registration.tmpl.cpp

```diff
@@ -37,7 +37,7 @@ namespace ${ns} {
 std::shared_ptr<Pothos::Block> factory__${factory.name}(${factory.exported_factory_args})
 {
     auto __orig_block = ${factory.factory_function_path}(${factory.internal_factory_args});
-    auto __pothos_block = makeGrPothosBlock(__orig_block);
+    auto __pothos_block = makeGrPothosBlock(__orig_block, ${"vlength" if "vlength" in factory.internal_factory_args else "vlen" if "vlen" in factory.internal_factory_args else 1}, ${"itemsize" if "const Pothos::DType &itemsize" in factory.exported_factory_args else "sizeof_stream_item" if "const Pothos::DType &sizeof_stream_item" in factory.exported_factory_args else "Pothos::DType()"});
```

Comment:
I recommend stuffing this logic as a member of factory and keep the template simple so its just `$factory.vlen`, etc 

right here is there the factory's params get stuffed in for template access, same logic can go there: https://github.com/pothosware/gr-pothos/blob/master/GrPothosUtil.py#L752

Reply:
Done.